common/OpTestLpar.py

```diff
@@ -285,3 +285,86 @@ def lpar_code_update(self, i_image, imagecomponent):
             print l_msg
             raise OpTestError(l_msg)
 
+    ##
+    # @brief It will check existence of given linux command(l_cmd) on lpar
+    #
+    # @param l_cmd @type string: linux command
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def lpar_check_command(self, l_cmd):
+        try:
+            l_res = self._ssh_execute(l_cmd)
```

Comment:
Should issue a "which l_cmd" and validate it exists that way.  Actually executing the command could have unintended consequences and does not match with the documentation for this interface.